src/components/Pagination/Pagination.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import type { Meta } from '@storybook/react';
+
+import Pagination, { PaginationProps } from './Pagination';
+// More on how to set up stories at: https://storybook.js.org/docs/react/writing-stories/introduction
+const meta = {
+  title: 'Components/Pagination',
+  component: Pagination,
+} satisfies Meta<typeof Pagination>;
+
+export default meta;
+
+interface StoryProps extends PaginationProps {
+  playgroundWidth: string;
+}
+
+const argTypes = {
+  disabled: {
+    control: {
+      type: 'boolean',
+    },
+  },
+  onChange: {
+    action: 'onChange',
+  },
+  onClick: {
+    action: 'onClick',
+  },
+  playgroundWidth: {
+    control: { type: 'range', min: 200, max: 300, step: 50 },
+  },
+  value: {
+    control: {
+      type: 'text',
+    },
+  },
+};
+
+export const Playground = ({ playgroundWidth, ...args }: StoryProps) => (
+  <div style={{ width: playgroundWidth, margin: '1rem' }}>
+    <Pagination key={args.value} {...args} id="Pagination-1">
+      <option>Lot 1</option>
```

Comment:
is there some way to enforce/document that the children need to be `<option>` here?

part of me wonders if an `options` prop might be simpler, but I guess it's not following the composable pattern as much

what do you think @scottdickerson